index.html

```diff
@@ -3787,6 +3787,16 @@ <h3>
 returned by this function into the <a href="#data-model">data model</a>.
                 </dd>
                 <dt>
+deactivated
+                </dt>
+                <dd>
+A <a data-cite="INFRA#boolean">boolean</a> value that, when set to true, indicates the <a>DID</a> supplied to the 
+<a>DID resolution</a> function has been deactivated and SHALL be recognized as 
+being in a deactivated state. This property is REQUIRED if the <a>DID</a> supplied 
```

Comment:
```suggestion
<a>DID resolution</a> function has been deactivated. This property 
is REQUIRED if the <a>DID</a> supplied 
```

It's not clear how "SHALL be recognized as being in a deactivated state" will be tested -- and feels duplicative of the next normative statement.